src/lib/fees/parameters/thor.ts

```diff
@@ -0,0 +1,19 @@
+import type { FeeCurveParameters } from './types'
+
+const FEE_CURVE_MAX_FEE_BPS = 0 // basis points
+const FEE_CURVE_MIN_FEE_BPS = 0 // basis points
+const FEE_CURVE_NO_FEE_THRESHOLD_USD = 0 // usd
+const FEE_CURVE_FOX_MAX_DISCOUNT_THRESHOLD = 0 // fox
+const FEE_CURVE_MIDPOINT_USD = 0 // usd
+const FEE_CURVE_STEEPNESS_K = 0 // unitless
+const FEE_CURVE_FOX_DISCOUNT_DELAY_HOURS = 0 // 7 for thorchain LP, see https://discord.com/channels/554694662431178782/920449863408287855/1207415430025580627
```

Comment:
nitpick: these comments are pretty irrelevant and the last one is unrelated if we just want to remove and have the single top level comment

Reply:
Done in [05daabc](https://github.com/shapeshift/web/pull/8044/commits/05daabc89d2082b788ce937e871349e89d2193ba)